tools/driver/swift_symbolgraph_extract_main.cpp

```diff
@@ -241,7 +241,29 @@ int swift_symbolgraph_extract_main(ArrayRef<const char *> Args, const char *Argv
 
   const auto &MainFile = M->getMainFile(FileUnitKind::SerializedAST);
   llvm::errs() << "Emitting symbol graph for module file: " << MainFile.getModuleDefiningPath() << '\n';
+  
+  int Success = symbolgraphgen::emitSymbolGraphForModule(M, Options);
+  
+  // Look for cross-import overlays that the given module imports.
+  
+  // Clear out the diagnostic printer before looking for cross-import overlay modules,
+  // since some SDK modules can cause errors in the getModuleByName() call. The call
+  // itself will properly return nullptr after this failure, so for our purposes we
+  // don't need to print these errors.
+  CI.getDiags().removeConsumer(DiagPrinter);
+  
+  for (const auto &ModuleName : VisibleModuleNames) {
+    if (ModuleName.str().startswith("_")) {
```

Comment:
I thought we had a helper for this, but apparently not, so now we have a starter bug suggesting adding one: https://bugs.swift.org/browse/SR-13953